core/include/tee/svc_cache.h

```diff
@@ -31,7 +31,8 @@
 #include <tee_api_types.h>
 
 #ifdef CFG_CACHE_API
-TEE_Result syscall_cache_operation(void *va, size_t len, unsigned long op);
+TEE_Result syscall_cache_operation(void *va, size_t len,
+				   enum utee_cache_operation op);
```

Comment:
I'd rather keep this as `unsigned long` instead of `enum utee_cache_operation` for two reasons, it's still `unsigned long` in `lib/libutee/include/utee_syscalls.h` and then there's also the comment at the beginning of that .h file.